install/migration/Makefile.am

```diff
@@ -2,9 +2,7 @@ NULL =
 
 appdir = $(IPA_DATA_DIR)/migration
 app_DATA =                              \
-	error.html			\
```

Comment:
This line is correct, but the patch is missing a removal of the `error.html` and `invalid.html` pages.

Reply:
What do you mean? Remove files upon upgrade/uninstall?